src/main/java/com/fasterxml/jackson/core/JsonParser.java

```diff
@@ -864,6 +864,18 @@ public void setCurrentValue(Object v) {
         assignCurrentValue(v);
     }
 
+    /**
+     * Get an approximate count of the number of tokens that have been read.
+     * This count is likely to be only updated if {@link StreamReadConstraints.Builder#maxTokenCount(long)}
+     * has been used to set a limit on the number of tokens that can be read.
+     *
+     * @return the number of tokens that have been read (-1 if the count is not available)
+     * @since 2.18
+     */
+    public long getTokenCount() {
```

Comment:
Actually, I think that I will change naming to `currentTokenCount()`, to follow new(er) naming convention we have going for, wrt dynamic values like this.